dotnet/src/SemanticKernel.Abstractions/Orchestration/IKernelExecutionContext.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.SemanticKernel.Orchestration;
+
+/// <summary>
+/// Kernel execution context
+/// </summary>
+public interface IKernelExecutionContext
+{
+    /// <summary>
+    /// App logger
+    /// </summary>
+    ILoggerFactory LoggerFactory { get; }
```

Comment:
Can we inject the logger factory directly into the components that need it?

Reply:
Not needed, removed